oem/ibm/configurations/pdr/ibm,bonnell/11.json

```diff
@@ -0,0 +1,420 @@
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/nvme3_identify",
+                "interface": "xyz.openbmc_project.Led.Group",
+                "property_name": "Asserted",
+                "property_type": "bool",
+                "property_values" : [false, true]
+             }
+        }]
+    },
+    {
+        "entity_path" : "/xyz/openbmc_project/inventory/system/chassis/motherboard/rdx0",
+        "effecters" : [{
+            "set" : { 
+                "id" : 17,
+                "size" : 1,
+                "states" : [1,2]
+            },  
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/rdx0_identify",
+                "interface": "xyz.openbmc_project.Led.Group",
+                "property_name": "Asserted",
+                "property_type": "bool",
+                "property_values" : [false, true]
+             }
+        }]
+    },
+    {
+        "entity_path" : "/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply0",
+        "effecters" : [{
+            "set" : {
+                "id" : 17,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/powersupply0_identify",
+                "interface": "xyz.openbmc_project.Led.Group",
+                "property_name": "Asserted",
+                "property_type": "bool",
+                "property_values" : [false, true]
+             }
+        }]
+    },
+    {
+        "entity_path" : "/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply1",
+        "effecters" : [{
+            "set" : {
+                "id" : 17,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/powersupply1_identify",
+                "interface": "xyz.openbmc_project.Led.Group",
+                "property_name": "Asserted",
+                "property_type": "bool",
+                "property_values" : [false, true]
+             }
+        }]
+    },
+    {
+        "type" : 45,
+        "instance" : 1,
+        "container" : 1,
+        "effecters" : [{
+            "set" : {
+                "id" : 10,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
+                "path": "/xyz/openbmc_project/led/groups/enclosure_fault",
+                "interface": "xyz.openbmc_project.Led.Group",
+                "property_name": "Asserted",
+                "property_type": "bool",
+                "property_values" : [false, true]
+             }
+        }]
+    },
+    {
+        "entity_path" : "/xyz/openbmc_project/inventory/system/chassis/motherboard/fan0",
+        "effecters" : [{
+            "set" : {
+                "id" : 10,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
```

Comment:
The dbus details are wrong in all the below paths in 11.json file.  Please check the 11.json file at "pldm/oem/ibm/configurations/pdr/ibm,rainier-1s4u" as an example.

So for fan0 it should be like below:
            "dbus" : {
                "path": "/xyz/openbmc_project/inventory/system/chassis/motherboard/fan0",
                "interface": "xyz.openbmc_project.State.Decorator.OperationalStatus",
                "property_name": "Functional",
                "property_type": "bool",
                "property_values" : [true, false]
             }